### ISSUE TITLE
Docs mention liveReloadHost. Should be liveReloadBaseUrl?

### ISSUE THREAD

**Issue by danwenzel** [ember-cli/ember-cli-inject-live-reload#52]
The README for this repo mentions a `liveReloadHost` option, but this doesn't seem to do anything, and I don't see it anywhere in the code. 

Should this be `liveReloadBaseUrl`?  That seems to be what actually changes the url of `ember-cli-live-reload.js`: https://github.com/ember-cli/ember-cli-inject-live-reload/blob/master/index.js#L41

**Comment by danwenzel** [comment]
@rwjblue - ⬆️ - Any insight here?  I'd be happy to open a PR.

**Comment by Redsandro** [comment]
@danwenzel I tried your suggestion and change the command line option, but I still cannot get it to work. Can you change these parameters using command line options at all?

See https://github.com/ember-cli/ember-cli-inject-live-reload/issues/53

**Comment by danwenzel** [comment]
@Redsandro - I don't think there's a way to do it via command line options or environment variables.  The idea is to add it to your `.ember-cli` file:

```
{
  "disableAnalytics": false,
  "usePods": true,
  "liveReloadBaseUrl": "http://localhost:4200/"
}
```

**Comment by Redsandro** [comment]
@danwenzel 

> I don't think there's a way to do it via command line options

There is, according to `ember help`:

```
$ ember help | grep reload
  --live-reload (Boolean) (Default: true)
  --live-reload-host (String) Defaults to host
  --live-reload-base-url (String) Defaults to baseURL
  --live-reload-port (Number) (Defaults to port number within [49152...65535])
```

I had trouble because of a firewall but now that I changed the configuration to allow the livereload port, I found that the `--live-reload-host` does actually work.

Analogous to that, I think that your premise is actually incorrect:

> The README for this repo mentions a `liveReloadHost` option, but this doesn't seem to do anything

The `liveReloadHost` option does exist. Keep in mind that command line options are dasherized versions of options; e.g.: `--live-reload-host`

I think the answer to this issue's title is _"no"_ and can safely be closed. Or change it in order to suggest better documentation, because there's quite some trial and error involved to substitute clear documentation.

I believe `host` is about the livereload websocket, and `base-url` is about the livereload javascript. I am still unsure.

See also: https://github.com/ember-cli/ember-cli-inject-live-reload/issues/33
